### DISCUSSION TITLE
Possibility to migrate from version 2 to version 3?

### DISCUSSION THREAD

**Discussion by uscreen-daniel** [requarks/wiki#4418] [Help / Questions]
Hello, is there already an information about the changes from version 2 to version 3? Is it possible to migrate the data? We are currently considering whether we should start with version 2 or wait for version 3.

Many thanks in advance.

**Comment by Kjenos** [comment]
I look for the same answer. I am especially interested in the migration to a new multi-site wiki. In our organisation there are multiple business connected, where the multi-site comes in handy. 

I'm not quite sure if its a good idea to already start with the wiki, if I have to reorganize later, after a potential V3 update.

**Comment by ieaster1** [comment]
Per this comment from Nicolas, you can see that v3 will be a seamless upgrade if you're using postgresql:
https://requarks.canny.io/wiki/p/import-from-v2

The offical word on release can be found on the release notes: https://docs.requarks.io/releases#dev-30
```
This build is under active development and has not yet been released.

ETA: Beta planned for Q4 2021 with final release in early 2022
```

If you plan this for a production environment, it looks like you have several months still until a final stable release will be available.
